WeakAurasOptions/AceGUI-Widgets/AceGUIWidget-WeakAurasAnchorButtons.lua

```diff
@@ -5,13 +5,15 @@ local AceGUI = LibStub and LibStub("AceGUI-3.0", true)
 if not AceGUI or (AceGUI:GetWidgetVersion(Type) or 0) >= Version then return end
```

Comment:
There's a bunch of mixed indentation. Some lines are indented by tabs, some by spaces